tests/functional/monitoring/prometheus/alerts/test_alert_mds_cpu_high_usage.py

```diff
@@ -27,12 +30,18 @@ def run_file_creator_io_with_cephfs(dc_pod_factory):
     interface = constants.CEPHFILESYSTEM
     log.info("Checking for Ceph Health OK")
     ceph_health_check_base()
+    ocp_project = OCP(
```

Comment:
as discussed, please create app pods on custom namespace and not in the openshift-storage namespace